VikingQuiz.Api/Controllers/QuizzesController.cs

```diff
@@ -109,82 +111,98 @@ public class QuizzesController : Controller
         [HttpPost("{id}")]
         [RequestSizeLimit(5_000_000)]
         [Authorize]
-        public async Task<IActionResult> Put(NewQuizViewModel quizBodyData, int id) {
+        public async Task<IActionResult> Put(NewQuizViewModel quizBodyData, int id)
```

Comment:
this method is too long. Try to split it in several smaller methods for better maintainability and readability